ion/tokenizer.go

```diff
@@ -1379,3 +1383,15 @@ func isNewLineChar(c int) bool {
 	return c == 0x0A || //new line
 		c == 0x0D //carriage return
 }
+
+func isClobWithInvalidChar(clob bool, c int) bool {
```

Comment:
Throughout: I suggest using `is____` for `bool` variable names.

Reply:
Would you rather just cutting ```is``` prefix from all the following functions:
```
isProhibitedControlChar()
isStringWhitespace() 
isNewLineChar()  
isClobWithNonAsciiChar()
```
(All these methods are added either in this PR or #58)

Reply:
I'm suggesting changing the boolean variable `clob` to `isClob` for readability. Removing the `is` from those functions seems to be going in the opposite direction?

Reply:
I thought you were talking about keeping ```is___``` only for variable names and not for function names.

I will updated the ```clob``` variable to ```isClob``` - I was following the existing ```clob``` variable.